app/assets/stylesheets/frontend.less

```diff
@@ -16,34 +16,43 @@ body {
 	font-family: sans-serif;
 }
 
-.section_headings {
-    position: relative;
-    z-index: 1;
-    overflow: hidden;
-    text-align: center;
+h1 {
+	/* Page itself has top padding */
```

Comment:
This theme is intended for general usage, and it can't be assumed there's only one h1 per page. We can add an "extra-stylesheets" argument to the html formatter if needed.